tsconfig.json

```diff
@@ -3,12 +3,18 @@
     "lib": ["es2015"],
     "target": "es5",
     "module": "commonjs",
-    "declaration": true,
+    "declaration": false,
```

Comment:
Why the change?

Reply:
The declaration setting is used to generate *.d.ts declaration files for your module. However, this setting does not work when you want to use .js files as a part of your code since TS won't know how to infer their module declaration. There's an issue/PR in TS to make this work while allowing for .js code to be apart of the repo (and ignored for type declarations). I think we'll be more likely to use some .js then to need to export types for this module.